app/ide-desktop/lib/dashboard/src/components/AriaComponents/Dialog/Dialog.tsx

```diff
@@ -32,7 +32,7 @@ const MODAL_STYLES = twv.tv({
 
 const DIALOG_STYLES = twv.tv({
   extend: variants.DIALOG_STYLES,
-  base: '',
+  base: 'text-xs text-primary',
```

Comment:
that's just the default size/color for the app. i assume this shouldn't be an issue in practice since it can be (or is) overridden by twMerge anyway right?

Reply:
i suppose the alternative is to add these two rules as default CSS for `.enso-dashboard` in `tailwind.css`?

Reply:
Not really, If it's default, then we probably should pull these classes up to the document level. Otherwise, it's strange if an element looks differently whether it's inside a dialog or not. Also, since our font letters are weirdly aligned inside the line, we had to use a custom(hacky, actually) approach to align the text in the grid(see: https://github.com/enso-org/enso/blob/develop/app/ide-desktop/lib/dashboard/src/components/AriaComponents/Text/Text.tsx#L38). 

Also note: since when we have the Text component, we no longer need to manually assign classes or something. Just instead of using `aria.Text` use `ariaComponents.Text`, or use TEXT_STYLES: https://github.com/enso-org/enso/pull/10087/files#diff-b2bf8fd6a3d76eb3615adda79e6a8d40d3e9dc3fd92d706aa2f24a9adda5377aR15

All in all, I think it's not a good idea to specify font size/color on a Dialog level.

(Small note - all texts have `text-primary/60` color)

In your scenario, it's best to either specify the text-size/color on the table level or use `ariaComponents.Text`

Reply:
> hacky approach

not sure this is needed - usually text uses `leading-snug` which follows the figma's approach to aligning the text

Reply:
> i suppose the alternative is to add these two rules as default CSS for .enso-dashboard in tailwind.css?

Yeah, probably, though this won't be a one-size-fits-all solution, because we still need to use TEXT_STYLES to properly align text in grid, see screenshot attached.

<img width="891" alt="image" src="https://github.com/enso-org/enso/assets/61194245/7c4d2859-e920-434d-8c0c-086c73ea3d0c">

Reply:
> all texts have `text-primary/60` color

note that `text-primary/60` *is* `text-primary`.

Reply:
Oh missed that, Thanks!